pilot/helpers/agents/Developer.py

```diff
@@ -324,48 +324,62 @@ def set_up_environment(self):
             }, FILTER_OS_TECHNOLOGIES)
 
         for technology in os_specific_technologies:
-            # TODO move the functions definitions to function_calls.py
-            cli_response, llm_response = self.convo_os_specific_tech.send_message('development/env_setup/install_next_technology.prompt',
-                { 'technology': technology}, {
-                    'definitions': [{
-                        'name': 'execute_command',
-                        'description': f'Executes a command that should check if {technology} is installed on the machine. ',
-                        'parameters': {
-                            'type': 'object',
-                            'properties': {
-                                'command': {
-                                    'type': 'string',
-                                    'description': f'Command that needs to be executed to check if {technology} is installed on the machine.',
-                                },
-                                'timeout': {
-                                    'type': 'number',
-                                    'description': 'Timeout in seconds for the approcimate time this command takes to finish.',
-                                }
-                            },
-                            'required': ['command', 'timeout'],
-                        },
-                    }],
-                    'functions': {
-                        'execute_command': execute_command_and_check_cli_response
-                    },
-                    'send_convo': True
-                })
+            llm_response = self.install_technology(technology)
 
+            # TODO: I don't think llm_response would ever be 'DONE'?
```

Comment:
you are right, once it is done it will return 'INSTALLED' which is defined in 'development/env_setup/install_next_technology.prompt'. This should be updated and maybe best way would be to put it in const/llm.py with other llm constants

Reply:
but also have in mind that this feature is disabled at the moment (return on line 312) so it is not important to fix it now